app/pages/patientdata/patientdata.js

```diff
@@ -488,6 +514,18 @@ export let PatientData = React.createClass({
     });
   },
 
+  handleSwitchToSettingsPrintView: function(e) {
+    this.props.trackMetric('Clicked Switch To Settings Print View', {
+      fromChart: this.state.chartType
+    });
+    if (e) {
+      e.preventDefault();
+    }
+    this.setState({
```

Comment:
[This is where I was recommending putting the `window.print()` call and dropping this `setState()`]